Rakefile.rb

```diff
@@ -67,3 +67,24 @@ def json_write(file, json)
     puts "Pull request #{pull_request.number} is outdated. (Newest pull request is #{pull_request.superseded_by.number})"
   end
 end
+
+desc "Post a summary of the pull request that's currently being built"
+task :pull_request_summary do
+  if ENV['TRAVIS_PULL_REQUEST'] == 'false'
```

Comment:
I think this would all be a lot cleaner if we had two separate tasks: one for travis to run, and one for humans to run.